zebra/zserv.c

```diff
@@ -1031,7 +1033,7 @@ void zserv_event(struct zserv *client, enum zserv_event event)
 {
 	switch (event) {
 	case ZSERV_ACCEPT:
-		event_add_read(zrouter.master, zserv_accept, NULL, zsock, NULL);
+		event_add_read(zrouter.master, zserv_accept, NULL, 0, NULL);
```

Comment:
hmm, does that work ? I think we need to include `zsock` there, so the select/poll knows when to call our callback?